frontend/src/components/SidebarPlugin/SideBar.vue

```diff
@@ -114,6 +111,11 @@ export default {
     logout() {
       this.$emit('logout')
     },
+    getElopageLink() {
+      return this.$store.state.hasElopage
+        ? `https://elopage.com/s/gradido/sign_in?locale=${this.$i18n.locale}`
+        : `https://elopage.com/s/gradido/basic-de/payment?locale=${this.$i18n.locale}&prid=111&pid=${this.$store.state.publisherId}&firstName=${this.$store.state.firstName}&lastName=${this.$store.state.lastName}&email=${this.$store.state.email}`
```

Comment:
Do we have to ensure a valid URI? What happens when first name is something like `Ana Maria`?

Reply:
For this I am not sure! I will make a try!